bin/reth/src/node/mod.rs

```diff
@@ -155,7 +155,8 @@ impl Command {
             network.clone(),
             TransportRpcModuleConfig::default()
                 .with_http(vec![RethRpcModule::Admin, RethRpcModule::Eth]),
-            RpcServerConfig::default().with_http(Default::default()),
+            RpcServerConfig::default()
+                .with_http(Default::default(), self.rpc.http_corsdomain.as_ref()),
```

Comment:
the corsdomain should be configured in a separate function.
with_http should only take the server config imo.